src/config/index.tsx

```diff
@@ -1,7 +1,12 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import "../utils/gaInit";
 import Root from "./components/Root";
 import store from "./store";
 
+ga("create", "UA-81509238-9", "auto");
+ga("set", "anonymizeIp", true);
+ga("send", "pageview", "config");
```

Comment:
Let's make third parameter look like the real url:

```suggestion
ga("send", "pageview", "/config.html");
```